share/cyclus.rng.in

```diff
@@ -1,202 +1,334 @@
+    <text/></element></optional>
+
+  <interleave>
 
-  <zeroOrMore>
-    <element name="commodity">
+    <optional><element name="ui"><text/></element></optional>
+    
+    <element name ="control">
+      <a:documentation>Block to define control parameters of simulation.</a:documentation>
       <interleave>
-        <element name="name"> <text/> </element>
-        <element name="solution_priority"> <data type="double"/> </element>
+        <optional>
+          <element name="simhandle">
+            <a:documentation>A user-defined simulation handle. Default is to generate a long random UUID.</a:documentation>
+            <data type="string"/> </element>
+        </optional>
+        <element name="duration"> 
+          <a:documentation>Simulation duration in units of time steps. Default time steps
+            are 1 month. See `dt` to change default time step length.</a:documentation>
+          <data type="nonNegativeInteger"/> </element>
+        <element name="startyear"> 
+          <a:documentation>First year of simulation, e.g. 2015 (integer)</a:documentation>
+          <data type="nonNegativeInteger"/> </element>
+        <element name="startmonth"> 
+          <a:documentation>First month of the simulation, e.g. 3 for March (integer)</a:documentation>
+          <data type="nonNegativeInteger"/> </element>
+        <optional>
+          <element name="decay"> 
+          <a:documentation>Mode for when radioactive decay occurs. Choose from "never", "manual", "lazy".</a:documentation>
+          <text/> </element>
+        </optional>
+        <optional> 
+          <element name="dt">
+            <a:documentation>Length of a time step in seconds. Default: 1 month = 1 year/12.</a:documentation>
+            <data type="nonNegativeInteger"/></element> 
+        </optional>
+        <optional>
+          <element name="explicit_inventory">
+            <a:documentation>A Boolean flag to indicate whether the database should include tables that explicitly
+              record the inventory of each resource buffer in each agent at each time step. (Default: False)</a:documentation>
+            <data type="boolean"/> </element>
+        </optional>
+        <optional>
+          <element name="explicit_inventory_compact">
+            <a:documentation>A Boolean flag to indicate whether the database should include compact tables that explicitly
+            record the inventory of each resource buffer in each agent at each time step. (Default: False)</a:documentation>
+            <data type="boolean"/> </element>
+        </optional>
+        <optional>
+            <element name="tolerance_generic">
+              <a:documentation>Value used as tolerance when comparing two generic floating point numbers. (Default: 1e-06)</a:documentation>
+              <data type="double"/></element>
+        </optional>
+        <optional>
+            <element name="tolerance_resource">
```

Comment:
An observation: I don't see these options listed on the website. We will want to visit if we want to add these to have the website be more complete.

Reply:
Yes - these and all the random number stuff that @nuclearkatie added!